DEVNOTES.txt

```diff
@@ -12,28 +12,26 @@ Further in our documentation, we always use the ./mvnw form. Please adapt it to
 
 Apache Ignite Maven Build Instructions
 ======================================
-1) Optional: build Apache Ignite.NET as described at modules/platforms/dotnet/DEVNOTES.txt.
-
-2) Compile and install:
+1) Compile and install:
 
         ./mvnw clean install -Pall-java,all-scala,licenses -DskipTests
 
-   or if you have built Apache Ignite.NET on the first step use following command:
-   (Note that 'doxygen' should be installed before running this command.)
+   or if you have to build .NET part as well, use the following command:
+   (Note that 'doxygen', '.NET Core SDK 3.1', 'PowerShell' should be installed before running this command.)
 
-        ./mvnw clean install -Pall-java,all-scala,licenses -DskipTests -DclientDocs
+        ./mvnw clean install -Pall-java,all-scala,licenses,platforms -DskipTests -DclientDocs
```

Comment:
I've tried `mvn clean package  -Pplatforms -DskipTests` and got `[WARNING] The requested profile "platforms" could not be activated because it does not exist.` - am I doing something wrong?

Reply:
The message you mentioned below appears due to the `ignite-dotnet` module itself doesn't contain any information about `platforms` profile, however we still need to add `ignite-dotnet` and `ignite-cpp` modules to the maven reactor.

If you're using the command below to build everything this message won't appear:
`mvn clean package -Pplatforms -DskipTests`

If you're using the command to build particular module, you'll see the message due to this module doesn't have any info about the `platforms` profile:
`mvn clean package -am -pl modules/platforms/dotnet -Pplatforms -DskipTests`